src/sql/workbench/browser/modelComponents/dropdown.component.ts

```diff
@@ -145,6 +145,7 @@ export default class DropDownComponent extends ComponentBase<azdata.DropDownProp
 			}
 			this._editableDropdown.enabled = this.enabled;
 			this._editableDropdown.fireOnTextChange = this.fireOnTextChange;
+			this._editableDropdown.input.setTooltip(!this.enabled ? this._editableDropdown.input.value : '');
```

Comment:
And did you test that this is updated correctly as the user types in the edit box?

Reply:
Oh if it's only for being disabled then ok. But you should only set this if the dropdown is disabled then.

Reply:
Since this only works for disabled dropdowns, user can't type. 
I originally tested this to add tooltip when the dropdown is disable. But if another interaction enables the dropdown, the tooltip needs to be removed via `setTooltip('')`.

Reply:
I can change it to first check if the input box has a tooltip. If the dropdown is now enabled, remove the tooltip. This will reduce the number of calls on setTooltip